### ISSUE TITLE
wsad for controlling camera

### ISSUE THREAD

**Issue by remembertoremind** [ihhub/fheroes2#3107]
Hi, wsad would be amazing. Would is be possible to add the abillity to change keys in a cfg? 

Love the project btw. Thank you _so_ much!

**Comment by ihhub** [comment]
Hi @remembertoremind , you could them even right now by modifying fheroes2.key file.

**Comment by remembertoremind** [comment]
> Hi @remembertoremind , you could change them even right now by modifying fheroes2.key file.

Omg I must be blind :D Thank you _so_ much!

**Comment by remembertoremind** [comment]
Hmm, I added the keys, but when starting the game the old keys are still in use. I also tried removing the keys like "s" for save game, but didn't help. Also tried to set a new key for save game, but it still uses the old key. Is the key-file supposed to be placed somewhere special? Atm I just have it in the same directory as the game.

`
scroll left              - (default keycode:  38 = 'a')
scroll right            - (default keycode:  40 = 'd')
scroll up               - (default keycode:  25 = 'w')
scroll down           - (default keycode:  39 = 's')
`
